.github/workflows/equinix_k8s_flow.yml

```diff
@@ -125,3 +127,24 @@ jobs:
           kubectl get nodes
           kubectl get pods --all-namespaces
  
+      - name: Pull kube-burner
+        run: |
+          git clone https://github.com/kube-burner/kube-burner
+          curl -sS -L "https://github.com/kube-burner/kube-burner/releases/download/v1.9.8/kube-burner-V1.9.8-linux-x86_64.tar.gz" | tar -xzC kube-burner/ kube-burner
+
+      - name: Port forward prom
+        run: |
+          kubectl port-forward -n monitoring svc/prometheus-k8s 9090:9090 &
+
+      - name: Generate metric-endpoint
+        run: |
+          cat > kube-burner/examples/workloads/kubelet-density/endpoints.yml <<EOF
+- endpoint: http://localhost:9090
```

Comment:
this probably will need indent to the same level as `cat`

Reply:
yeah, something is weird when I do indent this w/ pure bash.